packages/patterns/src/types.ts

```diff
@@ -779,9 +780,7 @@ export type CopyTaggedMethodGuard = CopyTagged<
  */
 export type MethodGuard = CopyTagged<'guard:methodGuard', MethodGuardPayload>;
 
-export type AwaitArgGuardPayload = {
-  argGuard: Pattern;
-};
+export type AwaitArgGuardPayload = { argGuard: Pattern };
```

Comment:
Not that it affects the review, but I am curious. Before and after looks like the same sequence of tokens (except for an optional semicolon). Why did `yarn format` change its mind?

Reply:
Not sure tbh, I will remove formatting changes